en/development/testing.rst

```diff
@@ -501,6 +501,10 @@ table linked to the model called Article. You can use any model available in
 your application. The statement will only import the Article schema, and  does
 not import records. To import records you can do the following::
 
+.. versionchanged:: 2.7 
+    ``model`` has been replaced by ``table``
```

Comment:
I'll take a look. Import with model should still work as there hasn't been any intentional changes to remove that functionality.